python/pocolog_convert.py

```diff
@@ -0,0 +1,31 @@
+import argparse
```

Comment:
Why do we need this file?
this is more or less a duplicate of convert.py

Reply:
yes, sorry I will delete this file. Historical reasons. I thought I already deleted it.